.github/workflows/commitint.yml

```diff
@@ -0,0 +1,23 @@
+name: commitlint
```

Comment:
your commit type is `build(ci)` but this should just be a `ci` type. maybe `ci(commitlint)` to distinguish it from the `ci.yml` file. `build` is for building the project, like if we added a compile step or something.

Reply:
I think ill make a commit template in the readme, should help some people out (me) I always forget what we have.

having links is nice for additional reading but i prefer a one stop shop